updateLocalesHelper/updateLocaleStatus.py

```diff
@@ -1,59 +1,92 @@
 #! /bin/env python
-import json
+from configparser import ConfigParser
+from http import HTTPStatus
+from json import dump
+from os import path, environ
+from requests import get as GET
```

Comment:
same here, I prefer to `import requests` and then `requests.get` - much more similar to their examples: https://requests.kennethreitz.org/en/master/user/quickstart/#make-a-request